commerce_fee.post_update.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Fee.
+ */
+
+/**
+ * Allows fee start and end dates to have a time component.
+ */
+function commerce_fee_post_update_1(array &$sandbox = NULL) {
```

Comment:
We shouldn't need this, since it's a "new" module." But I guess some may have had this installed before

Reply:
I'll be happy to remove it if you give me the word.  It came in handy for me because I had already installed the broken version in another environment, but now that the update has run I don't need it.